xml/pop_banners.xml

```diff
@@ -217,7 +217,58 @@ matched against these patterns to fingerprint POP3 servers.
     <param pos="0" name="service.product" value="Courier POP"/>
   </fingerprint>
   <!--
-
+<fingerprint pattern= "^CMailServer ([\d\.]+) POP3 Service Ready$">
+    <description>matches to CMailServer</description>
+    <example>CMailServer 5.0.0 POP3 Service Ready</example>
+	<param pos="0" name="service.product" value="CMailServer"/>
+	<param pos="0" name="service.vendor" value="Youngzsoft"/>
+	<param pos="0" name="os.family" value="Windows" />
+	<param pos="0" name="os.vendor" value="Microsoft"/>
+	<param pos= "1" name="service.version"/>
+  </fingerprint>
+<fingerprint pattern= "^POP3 Bigfoot v(\d\.\d) server ready$">
+    <description>matches to POP3 Bigfoot server</description>
+    <example>POP3 Bigfoot v1.0 server ready</example>
+    <param pos="0" name="service.vendor" value="Bigfoot"/>
+    <param pos="0" name="service.family" value="Bigfoot Email Tools"/>
+    <param pos="0" name="service.product" value="Bigfoot Email Tools"/>
+	<param pos="1" name="service.version"/>
+  </fingerprint>
+<fingerprint pattern="^(CCProxy) (\d\.\d) POP3 Service Ready$">
+    <description>matches the CCProxy POP3 server</description>
+    <example>POP3 Bigfoot v(1.0) server ready</example>
```

Comment:
This example doesn't match your fingerprint pattern. This is an example of where providing tests of the example's extractions as mentioned above would have caught this.